_sass/_search.scss

```diff
@@ -0,0 +1,20 @@
+/**
+ * Add styling used in search
+ */
+
+label[for="activity-search"] {
+    margin-bottom: 1em;
+    position: relative;
+}
+
+#clear-search {
```

Comment:
Yeah, I'm still searching for a way to have that positioned nicely in Firefox, Chromium and Opera …